Nodes/config/routes.php

```diff
@@ -14,17 +15,6 @@
     $routeBuilder->connect('/promoted/*', ['controller' => 'Nodes', 'action' => 'promoted']);
     $routeBuilder->connect('/search/*', ['controller' => 'Nodes', 'action' => 'search']);
 
-    $routeBuilder->scope('/:type', function (RouteBuilder $routeBuilder) {
-        $routeBuilder->routeClass('Croogo/Taxonomy.TypeRoute');
-        $routeBuilder->connect('/', ['controller' => 'Nodes', 'action' => 'index']);
-        $routeBuilder->connect('/archives/*', ['controller' => 'Nodes', 'action' => 'index']);
-        $routeBuilder->connect('/:slug', ['controller' => 'Nodes', 'action' => 'view']);
-        $routeBuilder->connect('/term/:slug/*', ['controller' => 'Nodes', 'action' => 'term']);
-    });
-
     // Content types
-    $routeBuilder->connect('/node', ['controller' => 'Nodes', 'action' => 'index']);
-    $routeBuilder->connect('/node/archives/*', ['controller' => 'Nodes', 'action' => 'index']);
-    $routeBuilder->connect('/node/:slug', ['controller' => 'Nodes', 'action' => 'view']);
-    $routeBuilder->connect('/node/term/:slug/*', ['controller' => 'Nodes', 'action' => 'term']);
+    CroogoRouter::routableContentTypes($routeBuilder);
```

Comment:
Hmm.

I wasn't able to get it to work, hence my original PR.  If we can do this without my patch, then we should do it.

Also, we can also do away with `routes=true` if you think that's better way.